vger/src/lib.rs

```diff
@@ -442,7 +442,12 @@ impl Renderer for VgerRenderer {
                     view: &texture_view,
                     resolve_target: None,
                     ops: wgpu::Operations {
-                        load: wgpu::LoadOp::Clear(wgpu::Color::WHITE),
+                        load: wgpu::LoadOp::Clear(wgpu::Color {
```

Comment:
maybe only do this if the window is transparent? also why a is 0.5 instead of 0.0?

Reply:
Is there such configuration? The transparency setting is for winit to signal to OS that it might be transparent.

> also why a is 0.5 instead of 0.0?

my own custom thing, will fix in a bit

Reply:
Yeah that's what I meant, when the WindowConfig has transparent setting in it.

Reply:
I have no idea how I'm supposed to do that, winit doesn't even expose the transparency state in Window, only in WindowBuilder